doc-tests/lib/travis.rb

```diff
@@ -0,0 +1,80 @@
+require "json"
```

Comment:
Should we use [this](https://github.com/travis-ci/travis.rb) library by travis to interact with travis?